src/SupervisionSuccess/model/calcRevocations.js

```diff
@@ -0,0 +1,39 @@
+const { exp } = Math;
+
+/**
+ * Calculates revocations
+ * @param month - Month
+ * @param IP - implementationPeriod (Interface -> A5)
+ * @param RA - revocations A (Parameters -> M)
+ * @param RT - revocations Timescale (Parameters -> K)
+ * @param changeInRevocations - integer value of revocations change
+ * @const RR - revocation reduction (Interface -> A3)
+ * @returns {number}
+ */
+function calcRevocations(month, IP, RA, RT, changeInRevocations) {
+  const RR = changeInRevocations * -0.01;
+
+  if (IP === 0) {
+    return RA * RT ** 2 * (1 - RR + RR * exp(-month / RT));
+  }
+
+  if (month > IP) {
```

Comment:
Month is just an integer representing a calendar month, between 1-12, right? And IP is a number of months? If so, this doesn't seem like the right comparison, because the month may be, say, 4, but be multiple years away from the initial month, and the IP could be 12. And so we'd be saying that month > IP but that's not contextually correct. Right? Or am I misunderstanding?

Reply:
If I'm wrong and month is actually the number of months since the initial month, then I think your logic is correct, but I would argue in favor of making the docstring more explicit about what the parameters actually are, given how dense the arithmetic is.

Reply:
@jessex month could be integer from 0 to Infinity and it represents quantity of months since the initial (month = 0).
Maybe i should rename this const, let me think.

Reply:
Calculations are right because tests have been ported from spreadsheet

Reply:
Added description to JSDoc